packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -335,7 +335,7 @@ export default function lwc(pluginOptions: RollupLwcOptions = {}): Plugin {
 
             if (warnings) {
                 for (const warning of warnings) {
-                    this.warn(transformWarningToRollupWarning(warning, src, filename));
+                    this.warn(transformWarningToRollupLog(warning, src, filename));
```

Comment:
I guess this is not a breaking change because we're passing our output back into Rollup itself rather than exposing it to customers.